crates/oxc_transformer/src/es2017/async_to_generator.rs

```diff
@@ -35,21 +47,18 @@
 //!
 //! Reference:
 //! * Babel docs: <https://babeljs.io/docs/en/babel-plugin-transform-async-to-generator>
-//! * Esbuild implementation: <https://github.com/evanw/esbuild/blob/main/internal/js_parser/js_parser_lower.go#L392>
 //! * Babel implementation: <https://github.com/babel/babel/blob/main/packages/babel-plugin-transform-async-to-generator>
-//! * Babel helper implementation: <https://github.com/babel/babel/blob/main/packages/babel-helper-remap-async-to-generator>
 //! * Async / Await TC39 proposal: <https://github.com/tc39/proposal-async-await>
 
-use oxc_ast::{
-    ast::{
-        ArrowFunctionExpression, Expression, Function, FunctionType, Statement,
-        VariableDeclarationKind,
-    },
-    NONE,
-};
-use oxc_semantic::ScopeFlags;
-use oxc_span::{GetSpan, SPAN};
-use oxc_traverse::{Ancestor, Traverse, TraverseCtx};
+use std::mem;
+
+use oxc_allocator::{Box, GetAddress};
+use oxc_ast::ast::*;
+use oxc_ast::Visit;
+use oxc_ast::NONE;
```

Comment:
Another style nit. Don't we usually go with this style?

```suggestion
use oxc_ast::{ast::*, Visit, NONE};
```